sdk/src/azure/core/az_aad.c

```diff
@@ -21,11 +21,8 @@ _az_aad_build_url(az_span url, az_span authority, az_span tenant_id, az_span* ou
   AZ_RETURN_IF_NOT_ENOUGH_SIZE(url, az_span_size(authority));
   az_span remainder = az_span_copy(url, authority);
 
-  {
-    int32_t url_length = 0;
-    AZ_RETURN_IF_FAILED(_az_span_url_encode(remainder, tenant_id, &url_length));
-    remainder = az_span_slice_to_end(remainder, url_length);
```

Comment:
With the changes from https://github.com/Azure/azure-sdk-for-c/pull/1022, we are no longer url encoding `tenant_id`. @vhvb1989 - can you fix this by reverting this part of the change, or is that not necessary?